tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -28,6 +41,48 @@ pub struct Layer<
 
 /// A builder for [`Layer`](struct.Layer.html) that logs formatted representations of `tracing`
 /// events and spans.
+///
+/// ## Examples
+///
+/// Constructing a layer with the default configuration:
+///
+/// ```rust
+/// # use tracing_subscriber::Layer as _;
+/// use tracing_subscriber::fmt;
+///
+/// let fmt_layer = fmt::Layer::builder().finish();
+/// # let subscriber = fmt_layer.with_subscriber(tracing_subscriber::registry::Registry::default());
+/// # tracing::subscriber::set_global_default(subscriber).unwrap();
```

Comment:
again, if we're going to construct a subscriber and set it as the default in the doctest, i'd rather that be part of the example; otherwise i would just remove this code.